### ISSUE TITLE
Terms Issue

### ISSUE THREAD

**Issue by CaptainSpike75** [nauvalazhar/my-login#20]
The Terms and Conditions link just takes you right back to the login page.

**Comment by bitress** [comment]
Right, but you need to make the Terms and Conditions page or maybe a modal for it to work.
